@navikt/core/react/src/modal/Modal.context.ts

```diff
@@ -4,6 +4,7 @@ import { createContext } from "../util/create-context";
 interface ModalContextProps {
   closeHandler?: React.MouseEventHandler<HTMLButtonElement>;
   ref: React.RefObject<HTMLDialogElement>;
+  size?: "small" | "medium";
```

Comment:
Since the `size`-prop only changes padding for header, body and footer we could omit having a context and solve it with some CSS
```css
.navds-modal--small .navds-modal__header {
...
}

.navds-modal--small .navds-modal__body {
...
}

.navds-modal--small .navds-modal__footer {
...
}
```